pybigtools/src/lib.rs

```diff
@@ -879,15 +879,34 @@ impl BigBedWrite {
 }
 
 enum BigWigAverageOverBedEntriesIteratorRet {
-    Single(f64),
-    WithName((String, f64)),
```

Comment:
This starts to seems a bit much. Given that we don't actually use this (it just gets immediately converted), seems to make sense to make `BigWigAverageOverBedEntriesIterator::__next__` just return a PyObject instead.